### PR TITLE
[Validator] Fix dockblock typehint in XmlFileLoader

### PR DESCRIPTION
**Pull request by tgalopin**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Validator/Mapping/Loader/XmlFileLoader.php [hunk 1]
```diff
@@ -185,8 +185,8 @@ protected function parseFile($path)
     /**
      * Loads the validation metadata from the given XML class description.
      *
-     * @param ClassMetadata $metadata         The metadata to load
-     * @param array         $classDescription The XML class description
+     * @param ClassMetadata     $metadata         The metadata to load
+     * @param \SimpleXMLElement $classDescription The XML class description
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Mapping/Loader/XmlFileLoader.php]: what about completely removing the doc block and adding the `\SimpleXMLElement` type hint to the signature itself?

**Comment by tgalopin** [comment]
Updated

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by xabbuh** [comment]
👍

**Comment by nicolas-grekas** [comment]
Thank you @tgalopin.

**Pull request merged**